__mocks__/handlers/self.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { rest } from 'msw';
+const URL = process.env.NEXT_PUBLIC_BASE_URL;
+
+export const userSelfData = {
+	id: 'aTOG168A86JXY5wVosJx',
+	github_display_name: 'Mahima Khandelwal',
+	github_id: 'Maheima',
+	roles: {
+		member: true,
+		archived: false,
+		super_user: false,
+	},
+	last_name: 'Khandelwal',
+	username: 'mahima',
+	incompleteUserDetails: false,
+	profileStatus: 'BLOCKED',
+	picture: {
+		publicId: 'profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s',
+		url: 'https://res.cloudinary.com/realdevsquad/image/upload/v1674639637/profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s.jpg'
+	},
+	status: 'active',
+	first_name: 'Mahima',
+	profileURL: 'https://mahima-profile-service.onrender.com'
```

Comment:
Please use dummy data for test

Reply:
done

Reply:
Where?

Reply:
That data is not created by me i refactored it for DRYing but i replaced my tests with duplicate data in path __tests__/Unit/hooks/tasksApi.test.tsx so no more tests depending on that data in my tests

Reply:
> Where?

That data is not created by me i refactored it for DRYing but i replaced my tests with duplicate data in path __tests__/Unit/hooks/tasksApi.test.tsx so no more tests depending on that data in my tests